checkstyle/checkstyle.xml

```diff
@@ -91,7 +91,7 @@
       <property name="allowEmptyConstructors" value="true"/>
       <property name="allowEmptyMethods" value="true"/>
     </module>
-    <module name="Indentation"/>
+    <!-- <module name="Indentation"/> -->
```

Comment:
@romani  this module is excluded for your convenience in order to suppress `Indentation` errors that are insignificant in this context (because we are hunting down only `UnnecessaryParentheses` regression).

Reply:
Ok, thanks.
But we will chase all Checks violations.

Reply:
Roger that @romani. 

Ok, I will revoke this module exclusion than (and then you will be able receive **_all_** Kafka's CheckStyle 8.42 violations). 

I will also rebase this PR onto trunk (CheckStyle upgrade from 8.20 to 8.36.2 was merged to trunk today).